packages/plugin-eslint/package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@code-pushup/utils": "0.45.0",
     "@code-pushup/models": "0.45.0",
-    "eslint": "^8.46.0",
+    "eslint": "~8.56.0",
```

Comment:
Again, keep the older version range, so that someone using ESLint <8.56.0 can still use Code PushUp.